src/test/java/core/basesyntax/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ParserServiceImplTest {
```

Comment:
rename tests in this class - parser should deal with transactions, not files, so - `validTransactions` `invalidTransactions` instead of emptyPath etc